docs/pages/clients/development-workflows.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Development Workflows
+---
+
+
+## Iterating on your product with custom clients
+
+With a custom client, you can provide a stable environment to load any changes to your application that can be defined in JavaScript or other asset files that must only be built and installed once. Other updates to your application, whether defined directly in your ios/ and android/ directories or by modules or SDKs you choose to install, will require a new version of your client to take advantage of your changes.
+
+To enforce an API contract between the JavaScript and native portion of your application, you should set the [`runtimeVersion`](../distribution/runtime-versions.md) value in your app.json or app.config.js. This runtime version will be used to ensure your clients only load compatible version of your application in both development and production.
```

Comment:
```suggestion
To enforce an API contract between the JavaScript and native portion of your application, you should set the [`runtimeVersion`](../distribution/runtime-versions.md) value in your `app.json` or `app.config.js`. This runtime version will be used to ensure your clients only load compatible versions of your application in both development and production.
```

"versions of your application" - I wonder if we can use a better word than 'application' here, since we're talking specifically about bundles, right? Application makes me think of the entire app

Reply:
Could we maybe be more direct about how this actually works? Something like
```suggestion
To enforce an API contract between the JavaScript and native portion of your application, you should set the [`runtimeVersion`](../distribution/runtime-versions.md) value in your `app.json` or `app.config.js`. Each build you make will have this value embedded and will only load bundles with the same `runtimeVersion`, in both development and production.
```
or maybe "updates" instead of "bundles"?